src/components/resource-section/add-evaluation/AddEvaluationAuthWidgets.js

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react';
+import { InputTextarea } from 'primereact/inputtextarea';
+import { Rating } from 'primereact/rating';
+import { Card } from 'primereact/card';
+import { Button } from 'primereact/button';
+import { Toast } from 'primereact/toast';
+import styles from './AddEvaluation.module.scss';
+import useLoginDialog from '@hooks/useLoginDialog';
+import { Tooltip } from 'primereact/tooltip';
+
+const AddEvaluationAuthWidgets = ({ formOptions }) => {
+  const [evaluation, setEvaluation] = useState(formOptions.evaluation);
+  const [comment, setComment] = useState(formOptions.comment);
+  const loginDialog = useLoginDialog();
+
+  useEffect(() => {
+    setEvaluation(formOptions.evaluation);
+  }, [formOptions]);
+
+  const handleErase = () => {
+    setComment('');
+    setEvaluation('');
+  };
+  let title = formOptions.evaluated ? 'Tu evaluación' : 'Agregar comentario';
+
+  const handleSubmit = () => {
+    formOptions.handleSubmitForm(evaluation, comment);
+  };
+
+  return (
+    <div
+      className="cardEvaluation"
+      onClick={() => {
+        !formOptions.enabled && loginDialog.setDisplayLoginDialog(true);
```

Comment:
I will approve but I think that this component can't know about the loginDialog.

Reply:
You are right, I pushed a new version that move `loginDialog` to a parent component.